webui/src/App.css

```diff
@@ -46,4 +46,14 @@
 .ValueMonitor-buttons,
 .ValueMonitor-label {
   flex: 0 0 auto;
+}
+
+.ToggleButton-plot-sensor {
+  margin-right: 0.75em;
+}
+
+.ToggleButton-plot-sensor > input {
```

Comment:
This is based on styles that the ToggleButtonGroup applies, but the ToggleButtonGroup had some other behavior I didn't want. I'm confused why the default ToggleButton type="checkbox" doesn't hide the checkbox element, from the docs it seems like it should.